client/src/components/organisms/Dashboard/AdminItems/AdminItems.js

```diff
@@ -63,13 +80,15 @@ export const AdminItems = () => {
       }
     });
 
-    const fetchTags = async () => {
-      const tags = await getTags(token);
-      setTags(tags);
-    };
+    // const fetchTags = async () => {
+    //   const tags = await getTags(token);
+    //   console.log('HAHAH', tags);
+    //   setAllTags(tags);
+    // };
```

Comment:
Im jsut getting rid of tags related stuff in this PR (maybe temporarily) as the associated queries are extremely slow and there seems to be very little use in e.g. filtering by tag names etc.